src/pybind/mgr/dashboard/plugins/ses7p_motd.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import hashlib
+import json
+
+from mgr_module import Option
+
+from . import PLUGIN_MANAGER as PM
+from . import interfaces as I  # noqa: E741,N812
+
+
+@PM.add_plugin  # pylint: disable=too-many-ancestors
+class SES7PMotd(I.CanMgr, I.Setupable, I.HasOptions):
+    @PM.add_hook
+    def get_options(self):
+        return [Option(
+            name='ses7p_motd_configured',
+            default=False,
+            type='bool',
+            desc='Is the MOTD for SES7P configured')]
```

Comment:
```suggestion
            desc='Enable the SES 7.1 message of the day')]
```
(nit)

Reply:
Done

Reply:
@votdev did you decide to go with the message as it was (`Is the MOTD for SES7P configured`)?

Reply:
Already done, see my comment above.

Reply:
I've modified the description according to Mikes suggestion (i like it more).